dll/ntdll/ldr/ldrutils.c

```diff
@@ -2257,7 +2257,7 @@ LdrpGetProcedureAddress(
     _In_ BOOLEAN ExecuteInit)
 {
     NTSTATUS Status = STATUS_SUCCESS;
-    UCHAR ImportBuffer[64];
+    UCHAR ImportBuffer[64]; // 128 since NT6.2
```

Comment:
Fortunately this initial size is more some kind of optimization, since, if it's too small, it gets reallocated (via the ImportName pointer).

Reply:
Yes, and also return a more reasonable error code if the reallocation failed since NT6.2